src/services/activity.js

```diff
@@ -149,7 +149,7 @@ export function formatMessage({
   // Check if find a value in the data (during transfers)
   if ('value' in dataCopy) {
     // Convert the value according to its denominator
-    const valueInCircles = formatCirclesValue(dataCopy.value);
+    const valueInCircles = formatCirclesValue(dataCopy.value, date);
```

Comment:
Is this the date of the transaction, right? i.e: https://github.com/CirclesUBI/circles-core/blob/main/src/activity.js#L129

Reply:
Have you checked that sometimes we need to use date.now() and sometimes the the when the tx happenes in the past?

Reply:
Yes as you can see in the code it is the same date as is displayed as transaction date in the activity log. Transactions from the core are provided from the core. 

In all other places I use now. The default is now. And it is only here for the activity log that i provide a date. I couldnt find other places that use formatCircleValues ie displays circles that are from the past